### PR TITLE
Update 9.0.0.sql

### PR DESCRIPTION
**Pull request by Kaikina**
See PR prestashop/prestashop#35843

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Goes along the PR prestashop/prestashop#35843 from prestashop/prestashop. A new configuration key is added to the core. It should be inserted in the next version autoupgrade script.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     |  NA
| Sponsor company   | Evolutive
| How to test?      | NA

### CODE REVIEW

**Comment by Kaikina** [comment]
Related PR : https://github.com/PrestaShop/PrestaShop/pull/35843

**Review by Touxten — APPROVED** [review]

**Review by LouisAUTHIE — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

**Comment by Hlavtox** [comment]
@Kaikina Rebase is needed

**Comment by Kaikina** [comment]
Done @Hlavtox

**Review by Hlavtox — CHANGES_REQUESTED** [review]
PR broken, some merge things got into it

**Comment by Kaikina** [comment]
@Hlavtox how about now ?

**Review by ga-devfront — APPROVED** [review]

**Review by Hlavtox — APPROVED** [review]

**Comment by AureRita** [comment]
Hi @Kaikina 

Currently I see that your [PR](https://github.com/PrestaShop/PrestaShop/pull/35843) isn't merged on develop, I'll block this PR until your PR 'll be merged 

Thank you

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
PR against the core has been merged. I removed the "Blocked" label.

**Comment by AureRita** [comment]
Hi @Kaikina !

Thank you for your PR, Currently I tried to see the difference with the 9.0.0 like if Ihave a new field to set up the new separators but as you can see, that doesn't seems to works : 

[recording(313).webm](https://github.com/user-attachments/assets/e97970e2-9964-4b15-be27-482aab5b7347)

image


Did I forget anything ? 

Waitinf gor your feedback

**Comment by jf-viguier** [comment]
@AureRita have you clone the core in the 9.0.x branch ? The other PR isn't in develop branch I guess ?

**Comment by Hlavtox** [comment]
@AureRita Why are you testing an already tested and merged feature?

Only thing that should be QAd now is if PS_SEPARATOR_FILE_MANAGER_SQL is in the database after upgrade.

**Comment by AureRita** [comment]
Hi @Kaikina 

Thank you for your PR, I tested it and it seems to works as you can see : 

image


Tested from : 
8.2 to 9.0.0
8.0.4 to 9.0.0

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Review by AureRita — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
